UnoCheck/Solutions/GTK3NotInstalledOnWindowsSolution.cs

```diff
@@ -0,0 +1,95 @@
+﻿using DotNetCheck.Models;
+using Polly;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Solutions
+{
+	public class GTK3NotInstalledOnWindowsSolution : Solution
+	{
+		public GTK3NotInstalledOnWindowsSolution(Uri url, string title)
+		{
+			Url = url;
+			Title = title;
+		}
+
+		public Uri Url { get; set; }
+		public string Title { get; set; }
+
+		public override async Task Implement(SharedState sharedState, CancellationToken cancelToken)
+		{
```

Comment:
Since we're duplicating this code, shouldn't this be refactored ? @jeromelaban

Reply:
Hello @jeromelaban, what do you want to do about that?

Reply:
Let's open an issue to take a look independently.

Reply:
I was created the issue #96